account-service/backend/app.py

```diff
@@ -0,0 +1,45 @@
+from flask import Flask, request, jsonify, session
```

Comment:
I thought we were using node.js as the language and express as the framework for our backend.
Flask is a web framework similar to express, but it's for Python.

Perhaps we should convert these to js, unless we want to change our tech stack to python?
Sorry for only feedbacking this now after you have done all these up already 🥲

Reply:
OH yep no worries!! i will convert them